analysis_options.yaml

```diff
@@ -153,7 +153,7 @@ linter:
     # - prefer_constructors_over_static_methods # far too many false positives
     - prefer_contains
     # - prefer_double_quotes # opposite of prefer_single_quotes
-    - prefer_equal_for_default_values
+    # - prefer_equal_for_default_values # Removed in Dart 3.0.0
```

Comment:
Just delete this line since it no longer appears in [1], which this file is based on (see note in line 33/34)?

https://github.com/dart-lang/linter/blob/main/example/all.yaml

Reply:
Ohh that makes sense, thanks!